driver_cpl/driver/map_lnd2rof_irrig_mod.F90

```diff
@@ -0,0 +1,287 @@
+module map_lnd2rof_irrig_mod
```

Comment:
My understanding is that driver_cpl/shr is for code needed by other components (as it is included in the share library, right?), whereas driver_cpl/driver is for code that is specific to the driver/coupler. This code is specific to the driver/coupler. In addition, since it has dependencies on seq_map_mod and seq_map_type_mod, which are in driver_cpl/driver, my impression is that it won't work to have this module in driver_cpl/shr. (My impression is that driver_cpl/shr can only depend on stuff in the share code library.)

But let me know if you feel differently on this.